src/components/CreateEncounter/GeneralInformation.js

```diff
@@ -0,0 +1,92 @@
+import HeadersComponent from 'components/common/HeadersComponent/HeadersComponent';
+import React, { useCallback } from 'react';
+import { Label } from 'reactstrap';
+
+import styled from 'styled-components/macro';
+import { format } from 'date-fns';
+
+import mobileIcon from 'assets/images/svg-icons/icon-phone.svg';
+import { GENDER_SHORTHAND } from '../../constants';
+import { handleCallAppointment } from 'utils';
+
+const Info = styled.div`
+  padding-left: 10px;
+  padding-top: 10px;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  color: #22335e;
+`;
+
+const Value = styled.div`
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  color: #22335e;
+  font-weight: 700;
+`;
+
+const GeneralInformation = ({ data, dispatch }) => {
+  const onCall = useCallback(
+    () => handleCallAppointment(dispatch, data?.patientId),
+    [dispatch, data],
```

Comment:
since only `patientId` is required inside the function, we can use `data.patientId` as dependency instead of the entire object

Reply:
The whole screen depends on multiple data fields.